grunt/tasks/options/jscs.coffee

```diff
@@ -0,0 +1,12 @@
+loadLocals = require('../../tools/loadLocals')
```

Comment:
The whole grunt config is in coffee. The comma free syntax make it easy to
read. It could be the next improvement to plan.

On Fri, 4 Mar 2016 at 14:24, Grégory Copin <notifications@github.com> wrote:

> In grunt/tasks/options/jscs.coffee
> <https://github.com/dummy-team/dummy/pull/86#discussion_r55028719>:
>
> > @@ -0,0 +1,12 @@
> > +loadLocals = require('../../tools/loadLocals')
>
> why keeping coffee conf if everything else is with ES6 ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dummy-team/dummy/pull/86/files#r55028719>.
>